src/pages/register/register.css

```
@@ -0,0 +1,195 @@
+header{
+    flex-direction: column;
+    justify-content: center !important;
+    gap: 1em;
+}
+
+label{
+    display: flex;
+    flex-direction: column;
+}
+input[type="text"]{
+    width: 18em !important;
+    padding-top: 0.3em;
+    padding-bottom: 0.3em;
+}
+input[type="date"]{
+    width: 8em !important;
+    padding-top: 0.25em;
+    padding-bottom: 0.25em;
+}
+input[type="number"]{
+    width: 2em !important;
+    padding-top: 0.35em;
+    padding-bottom: 0.35em;
+}
+select{
+    width: 13em !important;
+    padding-top: 0.3em !important;
+    padding-bottom: 0.3em !important;
+}
+
```

```suggestion
.icon-section[active] circle {
    fill: var(--purple);
}

.icon-section circle {
    fill: gray;
}
```

Aqui está a abordagem de utilização do svg para mudança de cor do círculo, baseado na presença ou não da propriedad `active` no elemento html